packages/contracts/deploy/verification/99_conclude/01_managing_dao_proposal.ts

```diff
@@ -0,0 +1,51 @@
+import {writeFile} from 'fs/promises';
+import {DeployFunction} from 'hardhat-deploy/types';
+import {HardhatRuntimeEnvironment} from 'hardhat/types';
+import {Multisig__factory} from '../../../typechain';
+import {getManagingDAOMultisigAddress, uploadToIPFS} from '../../helpers';
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  console.log('\nCreating managing DAO Proposal');
+  if (hre.managingDAOActions.length === 0) {
+    console.log('No actions defined');
+    return;
+  }
+
+  const {ethers, network} = hre;
+  const [deployer] = await ethers.getSigners();
+
+  const managingDAOMultisigAddress = await getManagingDAOMultisigAddress(hre);
+  const managingDAOMultisig = Multisig__factory.connect(
+    managingDAOMultisigAddress,
+    ethers.provider
+  );
+  const managingDAOMultisigSettings =
+    await managingDAOMultisig.callStatic.multisigSettings();
+
+  const proposalDescription = hre.managingDAOActions
+    .map(action => action.description)
+    .join('\n');
+  const cid = await uploadToIPFS(proposalDescription, network.name);
+
+  if (managingDAOMultisigSettings.onlyListed) {
+    if (!(await managingDAOMultisig.callStatic.isMember(deployer.address))) {
+      console.log(
+        `ManagingDAOMultisig (${managingDAOMultisigAddress}) doesn't allow deployer ${deployer.address} to create proposal.`
+      );
+      const tx = await managingDAOMultisig.populateTransaction.createProposal(
```

Comment:
Only multisig members are allowed to create a proposal in the managing DAO's multisig on mainnet, so this won't work.

https://app.aragon.org/#/daos/ethereum/management.dao.eth/settings

Reply:
Not entirely correct. On line 35 I use `populateTransaction` which returns an object with the parameters `to` and `data` defined (`from` too if we add a signer).
This is done in case the deployer is not a member of the Multisig so that we define the TX and write it to a file on line 45, which we then can send to a member of the Multisig to execute.

Reply:
Ah, so the intention is that someone sends the tx. Got it